### ISSUE TITLE
500 error

### ISSUE THREAD

**Issue by johnzastrow** [johnzastrow/choreme#5]
As parent, tried to list assigned chores. Displayed chores for today (1/2^), but paging to the next day triggered 500
![500error](https://user-images.githubusercontent.com/1213646/151266053-39b75f01-439b-471a-b28d-c0a0fbe358ac.png)
.

**Comment by johnzastrow** [comment]
Visiting the Rewards screen also throws 500.

**Comment by lamtuanvu** [comment]
Did you delete any chore from the document?

**Comment by johnzastrow** [comment]
Children's screens for chores assigned also throws 500

**Comment by johnzastrow** [comment]
Yes, to clean some up and to test deleting

**Comment by lamtuanvu** [comment]
Yes. That is the problem since we just delete chores but not their relations. Internal error happen when some other document expect the chore that was deleted already

**Comment by lamtuanvu** [comment]
Let me delete all tasks's collection then restart the recurring jobs
